who_is_active.sql

```diff
@@ -199,10 +207,28 @@ Formatted:		[context_switches] [varchar](30) NULL
 Non-Formatted:	[context_switches] [bigint] NULL
 	Shows the number of context switches, for active requests
 
-Formatted:		[used_memory] [varchar](30) NOT NULL

```

Comment:
Why is this nullable now? There is always a bit of used memory, even for sleeping connections.